.github/workflows/linux_tests.yml

```diff
@@ -92,4 +92,5 @@ jobs:
         shell: bash
         run: |
           export OMP_NUM_THREADS=1
-          pytest ${GITHUB_WORKSPACE}/tdms/tests/system/ -s -x  -m fdtd_build_only
\ No newline at end of file
+          pytest ${GITHUB_WORKSPACE}/tdms/tests/system/ -s -x  -m fdtd_build_only
+          
```

Comment:
```suggestion

```

Reply:
Oh thanks. Something's wrong with my vim config for yml.